openiddict/Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+    <PropertyGroup>
+        <CustomTargetFrameworks>net6.0;net8.0;</CustomTargetFrameworks>
```

Comment:
Essentially a global variable to use in other .csproj as `$(CustomTargetFrameworks)`